### ISSUE TITLE
window.phet.datamite doesn't follow convention for PhET globals

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/together#8]
datamite.js looks likes it's an example of how to use together.  It lives in the together repo, and creates global window.phet.datamite, which (according to https://github.com/phetsims/tasks/issues/160) doesn't follow PhET convention for naming globals.

Options:

(1) Allow this to be an exception to the convention. Explain why. 

(2) Rename to window.phet.together.datamite

(3) Move datamite to its own repo.

**Comment by samreid** [comment]
Since this is intended to be loaded by 3rd parties as a single *.js file library (like lodash, jquery or Pixi), perhaps it makes sense to export itself as a global (as all of those libraries do).  Thoughts?

**Comment by samreid** [comment]
Also, it is conceptually part of the together repo, since it is intended for (optional) use when together.js is used.

**Comment by pixelzoom** [comment]
> perhaps it makes sense to export itself as a global (as all of those libraries do)

That sounds reasonable.

**Comment by samreid** [comment]
So would you prefer window.datamite or window.phet.datamite?

**Comment by pixelzoom** [comment]
window.phet.datamite

**Comment by samreid** [comment]
I agree window.phet.datamite sounds good.  Anything else to do for this issue?  You reported in the original issue comment that it is already window.phet.datamite.
